eppo/src/main/java/cloud/eppo/android/EppoClient.java

```diff
@@ -354,6 +358,21 @@ public JSONObject getJSONAssignment(String flagKey, String subjectKey, JSONObjec
     return getJSONAssignment(flagKey, subjectKey, new SubjectAttributes(), defaultValue);
   }
 
+  public String getJSONStringAssignment(String flagKey, String subjectKey, String defaultValue) {
+    try {
+      EppoValue value =
+          this.getTypedAssignment(
+              flagKey,
+              subjectKey,
+              new SubjectAttributes(),
+              EppoValue.valueOf(defaultValue),
+              VariationType.JSON);
+      return value.stringValue();
```

Comment:
The only difference here is that we're not calling `parseJsonString` on the result of this method call